src/modules/access/common/WalletInterface.ts

```
@@ -50,22 +50,22 @@
     this.isAddress = false;
 
     if (!isPub) {
-      const _privateKey = Buffer.isBuffer(key) ? key : getBufferFromHex(key);
+      const _privateKey = ArrayBuffer.isView(key) ? key : hexToBytes(key as string);
       try {
-        if (_privateKey.length !== 32 || !isValidPrivate(_privateKey)) {
+        if (_privateKey.length !== 32 || !isValidPrivate(_privateKey as Uint8Array)) {
           throw new Error('Private key does not satisfy the curve requirements (ie. it is invalid)');
         }
       } catch (e) {
         throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)